src/combobox/index.ts

```diff
@@ -40,6 +40,7 @@ import { customElement } from '@dojo/framework/widget-core/decorators/customElem
  * @property onFocus            Called when the input is focused
  * @property onMenuChange       Called when menu visibility changes
  * @property onRequestResults   Called when results are shown; should be used to set `results`
+ * @property onResultSelect		Called when result is selected
```

Comment:
I'm such a nitpicker, but could you format this with spaces so it lines up? 🙇‍♀️